app/src/main/java/com/github/sdp/ratemyepfl/model/user/User.kt

```diff
@@ -11,24 +12,29 @@ data class User(
     val uid: String,
     val username: String?,
     val email: String?,
+    val picture: String? = "$uid.jpg",
```

Comment:
You can remove the picture field. And on the constructor just after, why are you putting null for the timetable ? Shouldn't you use an empty list ?

Reply:
Done.